tajo-common/src/main/java/org/apache/tajo/exception/TajoException.java

```diff
@@ -27,6 +28,16 @@
 public class TajoException extends Exception implements TajoExceptionInterface {
   private ResultCode code;
 
+  public TajoException(ReturnState e) {
```

Comment:
Stacktrace will be used in only errors like TajoInternalError because other exceptions mean user mistake and recoverable exceptions.